app/views/transferConfirmation.scala.html

```diff
@@ -0,0 +1,38 @@
+@()
+
+@main("Confirmation") {
+@defining(play.core.PlayVersion.current) { version =>
```

Comment:
Out of curiosity, what does this do? I can't see `version` being used anywhere.

Reply:
Not sure. Was in other templates, it can probably be removed